src/silx/gui/widgets/ElidedLabel.py

```diff
@@ -83,25 +83,33 @@ def __updateToolTip(self):
         else:
             qt.QLabel.setToolTip(self, self.__toolTip)
 
-    # Properties
+    # Inherited properties
+
+    def text(self):
+        """Returns the text defined by the user.
+
+        It can defier from the one really displayed, depending on the
+        `elideMode` defined for this widget.
+        """
+        return self.__text
 
     def setText(self, text):
         self.__text = text
         self.__updateText()
 
-    def getText(self):
```

Comment:
It would be better to keep `getText` for compatibility with a deprecation warning.